src/cli/cli.cpp

```diff
@@ -52,8 +52,8 @@
 #include <common/encoding.hpp>
 #include <common/new.hpp>
 #include <net/ip6.hpp>
-#include <platform/random.h>
 #include <platform/uart.h>
+#include <utils/global_address.hpp>
```

Comment:
Agree that non-core code should generally avoid including private includes from `src/core/*`.